src/platform/landing-pages/header-footer-data.json

```diff
@@ -0,0 +1,321 @@
+      "title": "Flickr",
+      "rel": "noopener noreferrer"
+    },
+    {
+      "column": 3,
+      "href": "https://www.youtube.com/user/DeptVetAffairs",
+      "order": 7,
+      "target": "_blank",
+      "title": "YouTube",
+      "rel": "noopener noreferrer"
+    },
+    {
+      "column": 3,
+      "href": "https://staging.va.gov/opa/socialmedia.asp",
+      "order": 8,
+      "target": null,
+      "title": "All VA social media"
+    },
+    {
+      "column": 4,
+      "href": "https://staging.va.gov/find-locations/",
+      "order": 1,
+      "target": null,
+      "title": "Find a VA location"
+    },
+    {
+      "column": 4,
+      "href": "https://iris.custhelp.va.gov/app/ask",
+      "order": 2,
+      "target": null,
+      "title": "Ask a question",
+      "rel": "noopener noreferrer"
+    },
+    {
+      "column": 4,
+      "label": "Call MyVA311:",
+      "href": "tel:18446982311",
+      "order": 3,
+      "target": null,
+      "title": "844-698-2311"
+    },
+    {
+      "column": 4,
+      "order": 4,
+      "title": "TTY: 711"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://www.section508.va.gov",
+      "order": 1,
+      "target": null,
+      "title": "Accessibility",
+      "rel": "noopener noreferrer"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/orm/NOFEAR_Select.asp",
+      "order": 2,
+      "target": null,
+      "title": "No FEAR Act Data"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/oig/",
+      "order": 3,
+      "target": null,
+      "title": "Office of Inspector General"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/opa/Plain_Language.asp",
+      "order": 4,
+      "target": null,
+      "title": "Plain language"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/privacy-policy/",
+      "order": 5,
+      "target": null,
+      "title": "Privacy, policies, and legal information"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/privacy/",
+      "order": 6,
+      "target": null,
+      "title": "VA Privacy Service"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/foia/",
+      "order": 7,
+      "target": null,
+      "title": "Freedom of Information Act (FOIA)"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://www.usa.gov/",
+      "order": 8,
+      "target": "_blank",
+      "title": "USA.gov",
+      "rel": "noopener noreferrer"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/scorecard/",
+      "order": 9,
+      "target": null,
+      "title": "VA.gov scorecard"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/veterans-portrait-project/",
+      "order": 10,
+      "target": null,
+      "title": "Veterans Portrait Project"
+    }
+  ],
```

Comment:
The footer links are just from https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/static-data/footer-links.json but exposed in `drupalHeaderFooter.json` for TeamSite pages to download